### ISSUE TITLE
contact info in bios

### ISSUE THREAD

**Issue by erikao** [mozilla/mozilla-opennews#240]
in addition to the general opennews email in the footer, our mozilla email addresses should be accessible via our bios on the site. i will update this later, but just wanted to log a ticket while i was thinking about it.

**Comment by sinker** [comment]
Yes, additionally, we should rig up a system where we add little author
bios to blog posts. w/contact info.
